appveyor.yml

```diff
@@ -18,8 +18,9 @@ build_script:
   - go build ./cmd/runhcs
   - go build ./cmd/tar2ext4
   - go build ./cmd/wclayer
-  - go build ./tools/uvmboot
-  - go build ./tools/zapdir
+  - go build ./internal/tools/uvmboot
+  - go build ./internal/tools/zapdir
+  - go build ./internal/tools/grantvmgroupaccess  
```

Comment:
Really dude. Why you gotta break my alphabetical ordering...

Reply:
You're having a really bad week, I can tell 😂